pages/gallery.html

```diff
@@ -1,12 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>gallery</title>
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/gallery.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+  </head>
+  <body>
+    <header>
+      <img class="floral-nav-divider" src="../assets/floral_white_smaller.png" alt="floral divider">
+      <div class="header-text-content">
+        <h1>Some photos about our favorite meals</h1>
+        <nav>
+          <ul>
+            <li><a href="../index.html">Home</a></li>
+            <li><a href="gallery.html">Gallery</a></li>
+            <li><a href="facts.html">Facts</a></li>
+          </ul>
+        </nav>
+      </div>
+      <img class="floral-nav-divider" src="../assets/floral_white_smaller.png" alt="floral divider">
+    </header>
+    <main>
+      <section class="gallery">
+        <h2>Enjoy these foods pictures!</h2>
+        <a class="slide-link tool-tip" href="#slide-1" tabindex="1">1
+          <span class="tool-tip-text">1: Cranberry Chicken</span>
+        </a>
+        <a class="slide-link tool-tip" href="#slide-2" tabindex="2">2
+          <span class="tool-tip-text">2: Blackened Fish</span>
+
+        </a>
+        <a class="slide-link tool-tip" href="#slide-3" tabindex="3">3
+          <span class="tool-tip-text">3: Beef Pho Noodle Soup</span>
+
+        </a>
+        <div class="slide-wrap">
+          <div class="slider">
+            <div class="slide" id="slide-1">
+              <article>
+                <img
+                src="../assets/foods-images/cranberry-chicken-2023-3.jpg"
+                alt="Food 1">
```

Comment:
The alt text for these images is not very descriptive, how could we update these to make the site more accessible?